examples/code/blend_modes/blend_modes.cpp

```diff
@@ -131,8 +131,8 @@ namespace
         pen::semaphore_post(p_thread_info->p_sem_continue, 1);
 
         pen::jobs_create_job(physics::physics_thread_main, 1024 * 10, nullptr, pen::e_thread_start_flags::detached);
-
-        put::dev_ui::init();
+        ImGuiStyle &style = ImGui::GetStyle();
```

Comment:
`put::dev_ui::init();` can handle this with the default argument, I expect to see no changes to this file in this pull request.